pkg/brokerapi/fake/server/convert_catalog.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	pkgbrokerapi "github.com/kubernetes-incubator/service-catalog/pkg/brokerapi"
+	"github.com/pivotal-cf/brokerapi"
+)
+
+// ConvertCatalog converts a (github.com/kubernetes-incubator/service-catalog/pkg/brokerapi).Catalog
+// to an array of brokerapi.Services
```

Comment:
I keep having to stop and think way too much about which brokerapi is which, up to you, but it might be nice to make ours "svccat" instead of "pkgbrokerapi" - so people don't need to think that much.